devp2p/tests/test_discovery.py

```diff
@@ -128,15 +129,15 @@ def test_ping_pong():
 
 eip8_packets = dict(
     # ping packet with version 4, additional list elements
-    ping1 = '''
+    ping1 = decode_hex(b'''
     e9614ccfd9fc3e74360018522d30e1419a143407ffcce748de3e22116b7e8dc92ff74788c0b6663a
     aa3d67d641936511c8f8d6ad8698b820a7cf9e1be7155e9a241f556658c55428ec0563514365799a
     4be2be5a685a80971ddcfa80cb422cdd0101ec04cb847f000001820cfa8215a8d790000000000000
     000000000000000000018208ae820d058443b9a3550102
-    '''.translate(None, ' \t\n').decode('hex'),
+    '''.replace(b' ', b'').replace(b'\n', b'')),
```

Comment:
I don't like these two `replace` calls here. Done it because `translate` works slight differently between Py2/Py3. Maybe creating an utility function like `remove_chars_from_str` or similar would be better?